package.json

```diff
@@ -79,6 +79,7 @@
   },
   "dependencies": {
     "iron-webcrypto": "^0.10.1",
-    "nanoid": "^5.0.1"
+    "nanoid": "^5.0.1",
+    "typescript": "^5.4.5"
```

Comment:
Can typescript be moved to devDependencies?

Reply:
Yeah that's where it should be